### DISCUSSION TITLE
[Help] Need to connect to DB through a Bastion server using DUO authorization

### DISCUSSION THREAD

**Discussion by hnavia** [dbeaver/dbeaver#21000] [Q&A] [answered]
I need to connect to a DB using SSH Tunneling. The problem is this:

1. I first connect to a bastion server
2. Here I receive a DUO push validation that I have to authorize from my phone
3. Once in the bastion, I need to enter "ssh server" being server the name of the server I want to connect

Once inside, I can get into the DB.

My question is, is it possible to use DBeaver to get into the DB using some sort of tunneling configuration?

Thank you in advance and kind regards.

**Comment by E1izabeth** [comment, marked as answer]
I hope this will answer your questions https://dbeaver.com/docs/dbeaver/SSH-Configuration/
